### DISCUSSION TITLE
Add PSAppDeployToolkit to the list of projects supplying an editorconfig file?

### DISCUSSION THREAD

**Discussion by mjr4077au** [editorconfig/editorconfig#529] [Wiki Edits]
Hi there!

I'm a developer of PSAppDeployToolkit, and we've been supplying a .editorconfigfile file for over 4 years now: https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/commit/557b337a0c3bf20e11a8ffaec026699140aa9f72

We wouldn't mind being added to the list of projects.

Thanks in advance 🤘

**Comment by xuhdev** [comment]
Done 👍
